aws_analysis_tools/cli/update_ec2_tags.py

```diff
@@ -107,8 +115,9 @@ def update_tags(
         classes = classes if classes is not None else self.classes
         dry_run = dry_run if dry_run is not None else self.dry_run
         environment = environment if environment is not None else self.environment
-        release = release if release is not None else self.release
         cluster_name = cluster_name if cluster_name is not None else self.cluster_name
+        lts = lts if lts is not None else self.lts
```

Comment:
I think it is fine on this use case, since I am using `if` as [ternary operator](http://www.dnawebagency.com/ternary-operator/). (Python [syntax](http://book.pythontips.com/en/latest/ternary_operators.html) is different from the standard notation with `?` and `:`.) Is this easy enough for you to read and understand what is happening? If not, I will change that.